gapic/samplegen/samplegen.py

```diff
@@ -118,15 +208,26 @@ class Validator:
     VAL_KWORD = "value"
     BODY_KWORD = "body"
 
-    def __init__(self, method: wrappers.Method):
-        # The response ($resp) variable is special and guaranteed to exist.
+    CHAIN_LINK_RE = re.compile(
```

Comment:
Why this name? Add a brief descriptive comment

Reply:
Think I'll change the name as well; the original impetus behind the name was that it was following the _chain_ of attributes within an expression, but it's not very helpful without that nugget of information.

Reply:
Going with EXPRESSION_ATTR_RE and adding a comment.

Reply:
Thanks; that's helpful! Could you put the comment right above the regex? It looks like it's describing `VALID_REQUEST_KWORDS` at the moment.

Reply:
Aaah, don't know how that happened.